packages/flutter/lib/src/rendering/paragraph.dart

```diff
@@ -257,9 +306,94 @@ class RenderParagraph extends RenderBox {
     return _textPainter.computeDistanceToActualBaseline(baseline);
   }
 
+  /// Intrinsics cannot be calculated without a full layout for
+  /// alignments that require the baseline.
+  bool _canComputeIntrinsics() {
+    for (PlaceholderSpan span in _placeholderSpans) {
+      if (span.alignment == ui.PlaceholderAlignment.baseline ||
+          span.alignment == ui.PlaceholderAlignment.aboveBaseline ||
+          span.alignment == ui.PlaceholderAlignment.belowBaseline) {
+        assert(RenderObject.debugCheckingIntrinsics, 'Intrinsics are invalid');
```

Comment:
"invalid" isn't useful to the developer. explain what is wrong and if possible how they can fix it.